### DISCUSSION TITLE
same method can only find one character

### DISCUSSION THREAD

**Discussion by billychen2014** [appium/appium#15447] [Q&A]
Hello, 

 i found one weird thing, please see the pic attached.

i can use the method   driver.find_element_by_ios_predicate(f"name CONTAINS '0'")   to find the element '0'   , but i can not find the element which value is '10002' in the same app and same page, hmm, is there something i did wrong ?

![image](https://user-images.githubusercontent.com/7981655/120288533-e7b1de80-c2f2-11eb-8309-a9162d64bbf2.png)
![image](https://user-images.githubusercontent.com/7981655/120288593-f8faeb00-c2f2-11eb-92a1-d03d5f94e745.png)

ENV:
device : iOS 13.6
Appium server:v1.18.1
thanks

**Comment by billychen2014** [comment]
i checked the page source , the element  below  can be  found  , which indicate that element exist

 <XCUIElementTypeStaticText type="XCUIElementTypeStaticText" value="10002" enabled="true" visible="true" x="0" y="88" width="414" height="22"/>

**Comment by jlipps** [comment]
Please make this an issue instead, and attach the requested logs, etc.
